properties/test_encode_decode.py

```diff
@@ -10,15 +10,10 @@
 
 import hypothesis.extra.numpy as npst
```

Comment:
These may need to be guarded too using `pytest.importorskip` perhaps? @max-sixty  what do you think?

Reply:
Yes same as here! https://github.com/max-sixty/xarray/blob/black/properties/test_encode_decode.py#L9

Reply:
Aha, I was being distracted by the other errors around the real one. Let's see if the latest commit helps.